src/test/java/com/hocs/test/pages/foi/SoftClose.java

```diff
@@ -0,0 +1,7 @@
+package com.hocs.test.pages.foi;
+
+import com.hocs.test.pages.decs.BasePage;
+
+public class SoftClose extends BasePage {
+
```

Comment:
same as above, are we missing something here or is this a placeholder for a later development?

Reply:
See other comment